cicd-scripts/app_start.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+echo "Activating virtual environment..."
+source /home/ec2-user/app/venv/bin/activate
+
+echo "Running searchgov-spider application..."
+# python /home/ec2-user/app/app.py
+
+pkill scrapy
+
+
+nohup scrapy crawl domain_spider &
+nohup scrapy crawl domain_spider_js &
+
+echo "\nCurrent running crawl jobs:"
+jobs
+
+echo "\noutput:"
+cat nohup.out
```

Comment:
do you think we will want to capture these logs in cloudwatch?  if so could stdout and stderr be redirected somewhere else?

Reply:
If so, im wondering if as a follow on we need some sort of logrotate.d or similar process to handle these.